client/translations/fi.json

```diff
@@ -1,66 +1,394 @@
+    "description": "Review line"
+  },
+  "EqGiDp": {
+    "defaultMessage": "Mitä kirjoitat tähän lomakkeen tulee näkymään julkisesti, joten älä kirjoita mitään, mitä et halua muiden näkevän. Et tarvitse kirjoittaa syy-kenttään mitään, jos et halua.",
+    "description": "Submission form list item 1"
+  },
+  "H65rvb": {
+    "defaultMessage": "Et ole tarpeeksi tyylikäs nähdääksesi tätä sivua.",
+    "description": "Error message shown when logged in but not having permission to view a page"
+  },
+  "HVzPRU": {
+    "defaultMessage": "Ei",
+    "description": "Poll results table header"
+  },
   "Hf1vPz": {
     "defaultMessage": "Valinnat",
     "description": "Nav link"
   },
+  "HvpI7B": {
+    "defaultMessage": "Poistettu beatmapsetti",
+    "description": "Placeholder for beatmapsets that were deleted from osu!"
+  },
+  "IiYTWb": {
+    "defaultMessage": "Nykyinen tiimi",
+    "description": "Team listing group title"
+  },
+  "IvOUKt": {
+    "defaultMessage": "<label>Pelimuoto:</label> {selector}",
+    "description": "Selector to change game mode"
+  },
+  "JPKVz4": {
+    "defaultMessage": "Beatmapsetti",
+    "description": "Mapper beatmapset consents table header"
+  },
+  "JPSkxB": {
+    "defaultMessage": "{country} lippu",
+    "description": "Country flag image alt text"
+  },
+  "Jy6JW8": {
+    "defaultMessage": "En tiedä kuka sinä olet! Yritä <a>kirjautua sisään</a> ensin.",
+    "description": "Error message shown when trying to view a protected page before logging in"
+  },
+  "LBBZpt": {
+    "defaultMessage": "Pelimuoto",
+    "description": "Poll results table header"
+  },
+  "Ll1LUO": {
+    "defaultMessage": "vahvasti puolesta",
+    "description": "+3 review score. Fits into {score} of the review line"
+  },
   "Lm++cZ": {
     "defaultMessage": "Project Loved",
     "description": "Site title"
   },
+  "MUDTXQ": {
+    "defaultMessage": "Beatmapsetin linkki tai ID",
+    "description": "Submission form beatmapset input prompt"
+  },
   "MoR8Of": {
-    "defaultMessage": "Mappaajien käyttöluvat",
+    "defaultMessage": "Mappaajien hyväksynnät",
     "description": "Nav link"
   },
+  "N4xf2i": {
+    "defaultMessage": "Suosikit",
+    "description": "Submissions table header"
+  },
+  "NG8X9U": {
+    "defaultMessage": "Parhaillaan rakenteilla",
+    "description": "Title of page not yet ready for public access"
+  },
+  "NjIKQe": {
+    "defaultMessage": "Ei saada yhteyttä",
+    "description": "Mapper consent shown in mapper consents table"
+  },
+  "O2IvSs": {
+    "defaultMessage": "{hasTimestamp,select,true{{user} ehdotti tämän mapin {timestamp, date, long}} other{{user} ehdotti tämän mapin}}",
+    "description": "Submission line"
+  },
+  "Oep6va": {
+    "defaultMessage": "Lähetä",
+    "description": "Submit button"
+  },
+  "P/DMly": {
+    "defaultMessage": "Pistemäärä <help>Väliaikainen tapa järjestää tätä listausta. Pistemäärä = Suosikkeja × 50 + Pelikertoja</help>",
```

Comment:
is it intentional that "Suosikkeja" doesn't match the header in the table?

if not, I was also thinking about changing this translation to read something like (fi for example):
"Pistemäärä \<help\>Väliaikainen tapa järjestää tätä listausta. {scoreDefinition}\</help\>"
and then I'd just use the translated table header names to build `scoreDefinition`.

Reply:
uh yeah they were supposed to match, thought i changed that